src/lib/components/workflow/dropdown-filter/workflow-type.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+  import DropdownMenu from '$lib/components/dropdown-menu.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
+  import { workflowFilters, workflowSorts } from '$lib/stores/filters';
+  import { updateQueryParamsFromFilter } from '$lib/utilities/query/to-list-workflow-filters';
+
+  let value = '';
+
+  const getOtherFilters = () =>
+    $workflowFilters.filter((f) => f.attribute !== 'WorkflowType');
+
+  $: typeFilter = $workflowFilters.find((f) => f.attribute === 'WorkflowType');
+  $: typeSort = $workflowSorts.find((s) => s.attribute === 'WorkflowType');
+
+  const onInput = (e: Event) => {
+    const { value } = e.target as HTMLInputElement;
+    if (value) {
+      const filter = {
+        attribute: 'WorkflowType',
+        value,
+        conditional: '=',
+        operator: '',
+        parenthesis: '',
+      };
+      $workflowFilters = [...getOtherFilters(), filter];
+    } else {
+      $workflowFilters = [...getOtherFilters()];
+    }
+
+    updateQueryParamsFromFilter($page.url, $workflowFilters, $workflowSorts);
+  };
+
+  function handleShowInput(event: CustomEvent) {
+    const show = event.detail.show;
+    if (show && typeFilter && !value) {
+      value = typeFilter.value;
+    }
+  }
+
+  function handleClearInput() {
+    $workflowFilters = [...getOtherFilters()];
+    updateQueryParamsFromFilter($page.url, $workflowFilters, $workflowSorts);
+  }
+</script>
+
+<DropdownMenu
+  value={typeFilter ? typeFilter.value : typeSort ? typeSort.value : ''}
```

Comment:
If I search for a workflow type > add other filters (e.g. start time) > navigate to a workflow > click the back button then the workflow type param still shows in search but not in the dropdown filter.

![Screen Shot 2022-10-12 at 10 13 34 AM](https://user-images.githubusercontent.com/15069288/195395359-1db3eff6-c459-47e9-879d-ec89aec83028.png)

When I add a workflow type back to the dropdown filter I get into an error state 👇 

![Screen Shot 2022-10-12 at 10 11 46 AM](https://user-images.githubusercontent.com/15069288/195395467-2428e3ff-50ca-4493-92d6-45ba4db02e3c.png)